curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-intermediate-algorithmic-thinking-by-building-a-dice-game/657e1bfa4f1f5d7d5c14d43d.md

```diff
@@ -0,0 +1,448 @@
+---
+id: 657e1bfa4f1f5d7d5c14d43d
+title: Step 76
+challengeType: 0
+dashedName: step-76
+---
+
+# --description--
+
+If the user did not get a `Full house` then the option for `None of the above` should be enabled with `0` points awarded. 
+
+Below your `if` statement, call the `updateRadioOption` function and pass in the numbers `5` and `0` for the arguments.
+
+# --hints--
+
+You should call the `updateRadioOption` function and pass in the numbers `5` and `0` for the arguments.
+
+```js
+assert.match(detectFullHouse.toString(), /updateRadioOption\s*\(\s*5\s*,\s*0\s*\)\s*;?/);
+```
```

Comment:
This should prevent people from adding `updateRadioOption(5, 0)` to the `if` statement, or somewhere else unexpected:

````suggestion
You should call the `updateRadioOption` function and pass in the numbers `5` and `0` for the arguments below your `if` statement.

```js
assert.match(code, /if\s*\(\s*hasThreeOfAKind\s*&&\s*hasPair\s*\)\s*{\s*updateRadioOption\s*\(\s*2\s*,\s*25\s*\)\s*;?\s*}\s*updateRadioOption\s*\(\s*5\s*,\s*0\s*\)\s*;?/);
```
````